samples/nrf9160/slm_shell/README.rst

```diff
@@ -0,0 +1,56 @@
+.. _slm_shell_sample:
+
+nRF9160: SLM Shell
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The SLM Shell sample demonstrates how to send AT commands to modem through the Serial LTE Modem (:ref:`slm_description`) running on nRF9160 SiP.
+This sample enables an external MCU to send modem or SLM proprietary AT commands for LTE connection and IP services.
+
+Requirements
+************
+
+SLM application should be configured to use UART_2 on nRF9160 side.
+
+The sample supports the following development kit:
+
+* PCA10056 nRF52840
+* PCA10095 nRF5340 (for the APP core only)
+
+Connect the DK with a nRF9160 DK based on the pin configuration in DTS overlay files of both sides.
+
+The following table shows how to connect PCA10056 UART_1 to nRF9160 UART_2 for communication through UART:
+
+.. list-table::
+   :align: center
+   :header-rows: 1
+
+   * - nRF52 Series DK
+     - nRF9160 DK
+   * - UART TX P1.02
+     - UART RX P0.11
+   * - UART RX P1.01
+     - UART TX P0.10
+   * - GPIO OUT P0.11 (Button 1)
+     - GPIO IN P0.31
+   * - GPIO IN P0.13 (optional)
+     - GPIO OUT P0.02 (optional)
```

Comment:
SLM use P0.02 as Indication pin for UART0 configuration.
Since we are using UART2 configuration, I'm not sure if we should use P0.31 as indication pin.
Or should we remove whole block of UART2 config in PR https://github.com/nrfconnect/sdk-nrf/pull/10049.

Reply:
P0.02 = LED1, just want to show the toggle of indication pin visually.
But it's all configurable.